File: 485.Max Consecutive Ones/solution.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0;
        int size = 0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]==1){
                size++;
                max = Math.max(max,size);
            }else{
                size=0;
            }
        }
        return max;
    }
}

File: 485.Max Consecutive Ones/solution.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        c = 0
        maxc = 0

        for i in nums:
            if i==1:
                c+=1
                maxc = max(maxc, c)
            else:
                c = 0
        return maxc
        